client-frameworks-support/client-support-angular/projects/client-support-angular/README.md

```diff
@@ -1,24 +1,112 @@
 # ClientSupportAngular
 
-This library was generated with [Angular CLI](https://github.com/angular/angular-cli) version 11.0.0.
+This library provides several features to run your angular application inside Luigi Micro-Frontend framework.  
+If you want to know more about Luigi Framework, please have a look on [Luigi Homepage](https://luigi-project.io/).
 
-## Code scaffolding
+## How to use the library
+Using the library is pretty straightforward, you need to import the library in the package.json:
+```javascript
+npm install @luigi-project/fundamental-ngx-sample-apps-master -s
+```
 
-Run `ng generate component component-name --project client-support-angular` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module --project client-support-angular`.
-> Note: Don't forget to add `--project client-support-angular` or else it will be added to the default project in your `angular.json` file. 
+Once the library is imported and saved in your angular project, you will need to import the module LuigiAngularSupportModule:
 
-## Build
+```javascript
+imports: [
+  ........
+  ,LuigiAngularSupportModule
+],
+```
 
-Run `ng build client-support-angular` to build the project. The build artifacts will be stored in the `dist/` directory.
+## Features
+Here the main features provided by the library:
+* LuigiContextService
+* LuigiAutoRoutingService
 
-## Publishing
+### LuigiContextService
+You can inject this service inside your Angular items in order to:
+* Get the current Context (latest) that we received from Luigi Core
+* Provide an Observable<Context> where through subscribing, you can get any Context change     
+    
+LuigiContextService is an abstract class, its implementation is in LuigiContextServiceImpl class.  
+If you need to change/extend the implementation, you can easily create your a new class extending LuigiContextServiceImpl:
 
-After building your library with `ng build client-support-angular`, go to the dist folder `cd dist/client-support-angular` and run `npm publish`.
+```javascript
+export class YourContextService extends  LuigiContextServiceImpl {
+    ....    
+}
 
-## Running unit tests
+```
+Inside your module redefine the provider:
+ ```javascript
+providers: [
+    {
+        provide: LuigiContextService,
+        useClass: YourContextService
+    }
+]
+ ```
+    
+### LuigiAutoRoutingService
+This service you cannot directly be used, but it will provide useful features how to synchronize your angular application with Luigi navigation.  
+It can happen that in your microfrontend, user can navigate through different components/pages.  
+With this feature we provide an easy way how to synchronize angular route with Luigi navigation; in angular route configuration, you can now add in data these attributes:
```

Comment:
```suggestion
This service cannot be referenced directly, but it will provide useful features for synchronizing your Angular application with Luigi navigation.

If the user navigates to different components or pages inside the micro frontend, this feature allows you to detect it and synchronize the Angular route with your Luigi navigation. In the Angular route configuration, you can now add these attributes to the data:
```

Maybe we can elaborate on what it means that the feature cannot be used directly. Does it mean it can still be used, but not referenced directly? Because if it cannot be used, then why do we have it?